cmd/kubeadm/app/constants/constants.go

```diff
@@ -344,7 +344,7 @@ const (
 	KubeDNSVersion = "1.14.13"
 
 	// CoreDNSVersion is the version of CoreDNS to be deployed if it is used
-	CoreDNSVersion = "1.7.0"
+	CoreDNSVersion = "v1.8.0"
```

Comment:
just noting that if one day we want to *not* use a certain version of coredns with a version of k8s, we may have to introduce the same mapping / table as the one for etcd on line 448.